birdhouse/scripts/logging.include.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+BIRDHOUSE_COLOR=${BIRDHOUSE_COLOR:-1}
+if [ "${BIRDHOUSE_COLOR}" -eq "1" ]; then
+    BLUE=$(tput setaf 12)
+    GRAY=$(tput setaf 8)
+    CYAN=$(tput setaf 6)
```

Comment:
Why add in these unused colours as well?

Reply:
`MSG_[...]` can be overridden with other sets of colors if desired. I just left them there for convenience instead of looking up the values each time.